pkg/locodedb/calls_test.go

```diff
@@ -19,7 +19,7 @@ func TestGet(t *testing.T) {
 	})
 
 	t.Run("locode", func(t *testing.T) {
-		rec, err := locodedb.Get("RUMOW")
+		rec, err := locodedb.Get("RU MOW")
```

Comment:
the commit says "as well as", so lets have both tests?

Reply:
now we have one test with  space "RU MOW" and 5 more without space will be added in https://github.com/nspcc-dev/locode-db/pull/23